CMakeLists.txt

```diff
@@ -117,4 +117,19 @@ if (SIMSIMD_BUILD_SHARED)
     add_library(simsimd_shared SHARED ${SIMSIMD_SOURCES})
     target_include_directories(simsimd_shared PUBLIC "${PROJECT_SOURCE_DIR}/include")
     set_target_properties(simsimd_shared PROPERTIES OUTPUT_NAME simsimd)
+
+    install(
+        TARGETS simsimd_shared
+        ARCHIVE
+        BUNDLE CXX_MODULES_BMI
+        FRAMEWORK
+        LIBRARY
+        OBJECTS
+        PRIVATE_HEADER
+        PUBLIC_HEADER
+        RESOURCE
+        RUNTIME)
 endif ()
+
+install(DIRECTORY ./include/ DESTINATION /usr/include/)
+install(DIRECTORY ./c/ DESTINATION /usr/src/${PROJECT_NAME}/)
```

Comment:
Is it a good practice to include the sources in the installation?

Reply:
Nothing unusual. See the package contents [here](https://archlinux.org/packages/extra/x86_64/gtest/).

It is required by usearch, https://github.com/unum-cloud/usearch/blob/e7140e55967a4198f2bbd34987fa7597ebafa649/c/CMakeLists.txt#L5.